SimonImanigirimpuhwe/my-brand-website
# #13 Ft update profile information #13

#### What does this PR do?

- Enable admin to change profile information

#### Description of Task to be completed?

- Make a form for input changes

- Add scripts to display profile edit form

- Add scripts to sign-out from the dashboard

#### How should this be manually tested?

- login in to get redirected to the dashboard, you will see a profile setting display

- Click on profile you should be able to see the admin profile

- Then click on profile, immediately the edit profile form will display

#### Any background context you want to provide?

- The admin dashboard page is created at the time of registration

#### What are the relevant Trello stories?

- [#13](https://trello.com/c/T5kEKsj3/13-update-profile-information)

#### Screenshots (if appropriate)

![update-profile-screenshoot](https://user-images.githubusercontent.com/54619678/93762763-791f6b80-fc10-11ea-9bd0-99f98adb701f.png)



#### Questions:

- N/A
